bundles/pubsub/pubsub_utils/src/pubsub_skt_handler.c

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 /*
- * pubsub_tcp_handler.c
+ * pubsub_skt_handler.c
```

Comment:
pubsub has two libs pubsub_utils and pubsub_spi.

Although I think there are not ideally split, the idea is that:
 - pubsub_spi should be used/needed when you want to extend the PSA system (new PubSubAdmin or PubSubDiscovery)
 - pubsub_utils should be used/needed when you use the PSA system (Add your own interceptor, etc)

So I think the pubsub_skt_handler can go to the pubsub_spi lib.